### ISSUE TITLE
Localisation: Get AppWrite errors in users Locale. 

### ISSUE THREAD

**Issue by vykes-mac** [appwrite/appwrite#4876]
This issue is more of a question. Does AppWrite return errors in the users locale?

I see a config [here](https://github.com/appwrite/appwrite/blob/a983630e0e6a38e88e5738801057dec5a9ad6c43/app/config/errors.php) with error types and messages, However the error messages seem to be hardcoded and not read from a translation file.

Example: 
```php
Exception::USER_INVALID_CREDENTIALS => [
        'name' => Exception::USER_INVALID_CREDENTIALS,
        'description' => 'Invalid credentials. Please check the email and password.',
        'code' => 401,
    ],
```

Even when I set the user's locale with `setLocale` api, it still return the above error message in plain english.

Thank you for your response!

**Comment by stnguyen90** [comment]
@vykes-mac thanks for your suggestion! 🙏🏼 We'll keep this in mind for the future. For now, you might need to add translations yourself. You can use the `type` of an Apwrite error to decide what error to show. See our list of [Error Types](https://appwrite.io/docs/response-codes#errorTypes).

**Comment by stnguyen90** [comment]
@vykes-mac actually, is this a duplicate of https://github.com/appwrite/appwrite/issues/1478? If so, would you please 👍🏼 that issue and close this issue?

**Comment by vykes-mac** [comment]
@stnguyen90 thank you for your response.

**Comment by rorellanamakingsense** [comment]
what does it mean **_To switch locales, all you need to do is pass the 'X-Appwrite-Locale' header or set the 'setLocale'_** in [the locale docs](https://appwrite.io/docs/client/locale) ?
